conans/server/rest/controllers/conan_controller.py

```diff
@@ -15,7 +15,7 @@ class ConanController(Controller):
     """
     def attach_to(self, app):
 
-        conan_route = '%s/:conanname/:version/:username/:channel' % self.route
+        conan_route = '%s/<conanname>/<version>/<username>/<channel>' % self.route
```

Comment:
The ":var" way to specify parameters was deprecated a long time ago. Not related with the revisions feature.